topi/python/topi/cuda/dense.py

```diff
@@ -135,6 +131,127 @@ def traverse(OP):
     return s
 
 
+def schedule_dense_small_batch(cfg, s, C, outs):
+    """Schedule float32/64 dense with small batch size"""
+    A, _ = C.op.input_tensors
+    _, in_dim = get_const_tuple(A.shape)
+    cfg.define_split('tile_k', in_dim, num_outputs=2)
+    if cfg.is_fallback:
+        cfg["tile_k"] = SplitEntity([-1, 64] if in_dim > 64 else [1, 64])
+
+    _, kf = cfg['tile_k'].apply(s, C, C.op.reduce_axis[0])
+    CF = s.rfactor(C, kf)
+
+    if C.op in s.outputs:
+        Out = C
+    else:
+        Out = outs[0].op.output(0)
+        s[C].compute_at(s[Out], s[Out].op.axis[1])
+    s[Out].bind(s[Out].op.axis[0], tvm.thread_axis("blockIdx.y"))
+    s[Out].bind(s[Out].op.axis[1], tvm.thread_axis("blockIdx.x"))
+
+    tx = s[C].op.reduce_axis[0]
+    thread_x = tvm.thread_axis("threadIdx.x")
+    s[C].bind(tx, thread_x)
+    s[CF].compute_at(s[C], tx)
+    s[C].set_store_predicate(thread_x.var.equal(0))
+    s[Out].set_store_predicate(thread_x.var.equal(0))
+
+def schedule_dense_large_batch(cfg, s, C):
```

Comment:
Does schedule_dense_large_batch support operator fusion to handle bias? Seems not. It doesn't have outs in the argument list.

Reply:
Also, we should add a large-batch test case to https://github.com/dmlc/tvm/blob/master/topi/tests/python/test_topi_dense.py#L117

Reply:
Thanks for pointing out. Fixed.